logs.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+const debug = require('debug')
+
+class Logger {
+  constructor (namespace) {
+    this.debug = debug(namespace)
```

Comment:
maybe we should separate the log and trace levels here, by doing something like

```
this.debug = debug(namespace)
this.trace = debug(namespace + ':trace')
```

That way the plugin has one logger object. When a riverpig object is passed in we can just pass through debug calls to it. For traces, I'm less sure what to do. We may need to do create a new debug based off of the riverpig logger's namespace by calling `debug(logger.debug.namespace + ':debug')`